### PR TITLE
CATROID-677 Make "No internet" message easier to read

### PR DESCRIPTION
**Pull request by frit96**
https://jira.catrob.at/browse/CATROID-677
Make "No internet" message easier to read

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by adnaribo — CHANGES_REQUESTED** [review]
Unfortunately the same happens as in https://github.com/Catrobat/Catroid/pull/3739 See screenshot below
Screenshot from 2021-07-07 22-30-11

File: catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java [hunk 1]
```diff
@@ -208,6 +208,19 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
 			if (Utils.checkIsNetworkAvailableAndShowErrorMessage(WebViewActivity.this)) {
 				ToastUtil.showError(getBaseContext(), R.string.error_unknown_error);
 			}
+
+			if (errorCode == ERROR_CONNECT
+					|| errorCode == ERROR_FILE_NOT_FOUND
+					|| errorCode == ERROR_HOST_LOOKUP
+					|| errorCode == ERROR_TIMEOUT
+					|| errorCode == ERROR_UNKNOWN) {
```
Comment (hofcle78) [inline catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java:216]:
```suggestion
					|| errorCode == ERROR_UNKNOWN),
					|| errorCode == ERROR_PROXY_AUTHENTICATION) {
```
adding **ERROR_PROXY_AUTHENTICATION** should fix the problem which occurs on BrowserStack devices.
with this changes i have tested it on:
- Galaxy S9 Android 10 (real device)
- Pixel 4 Android 11 (Android Studio)
- Pixel 3 Android 9 (BrowserStack)
- GalaxyTab S5e (BrowserStack)
image

File: catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java [hunk 2]
```diff
@@ -208,6 +208,19 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
...
+				setContentView(R.layout.activity_network_error);
+				setSupportActionBar(findViewById(R.id.toolbar));
+				getSupportActionBar().setIcon(R.drawable.pc_toolbar_icon);
+				getSupportActionBar().setTitle(R.string.app_name);
+			} else {
+				Log.e(TAG, "could'nt connect to the server! info: " + description + " : " + errorCode);
```
Comment (hofcle78) [inline catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java:222]:
```suggestion
				Log.e(TAG, "couldn't connect to the server! info: " + description + " : " + errorCode);
```

**Review by hofcle78 — CHANGES_REQUESTED** [review]

**Comment by frit96** [comment]
Thank you for the review I made all the requested changes

**Review by hofcle78 — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**